requirements.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
MAGStock doesn't currently have any Python dependencies above and beyond what Uber already has, but if it did then we could define them here.